src/storage/src/hummock/event_handler/hummock_event_handler.rs

```diff
@@ -427,8 +437,12 @@ impl HummockEventHandler {
 
         let new_pinned_version = pinned_version.new_pin_version(newly_pinned_version);
 
-        self.refiller
-            .start_cache_refill(sst_delta_infos, pinned_version, new_pinned_version);
+        self.refiller.start_cache_refill(
```

Comment:
Not related to this PR: should we add a check here to skip `start_cache_refill` and apply version directly if `sst_delta_infos` is empty to avoid spawning noop tokio task? This will break the apply order of CN flush version delta and compaction version delta though.